packages/web/app/components/Medication/MedicationTable.jsx

```diff
@@ -147,116 +147,165 @@ const getFrequency = ({ frequency, encounterPrescription, discontinued }, getTra
   );
 };
 
-const MEDICATION_COLUMNS = (getTranslation, getEnumTranslation, disableTooltip) => [
-  {
-    key: 'medication.name',
-    title: <TranslatedText stringId="medication.table.column.medication" fallback="Medication" />,
-    accessor: data => getMedicationName(data, getEnumTranslation),
-    CellComponent: props => <LimitedLinesCell {...props} disableTooltip={disableTooltip} />,
-  },
-  {
-    key: 'dose',
-    title: <TranslatedText stringId="medication.table.column.dose" fallback="Dose" />,
-    accessor: data => {
-      const pauseData = data.encounterPrescription?.pausePrescriptions?.[0];
-      const isPausing = !!pauseData && !data.discontinued;
-      return (
-        <NoWrapCell
-          color={isPausing ? Colors.softText : 'inherit'}
-          fontStyle={isPausing ? 'italic' : 'normal'}
-        >
-          {getMedicationDoseDisplay(data, getTranslation, getEnumTranslation)}
-          {data.isPrn && ` ${getTranslation('medication.table.prn', 'PRN')}`}
-        </NoWrapCell>
-      );
+const MEDICATION_COLUMNS = (
```

Comment:
this is a nit pick and up to you if you want to change it but I would prefer this to be camel case medicationColumns since it's a function

Reply:
Oh yep totally agree :+1: